detectors/detector.py

```diff
@@ -14,6 +14,8 @@ def get_bounding_boxes(frame, model):
         from detectors.tfoda import get_bounding_boxes as gbb
     elif model == 'detectron2':
         from detectors.detectron2 import get_bounding_boxes as gbb
+    elif model == 'tfoda_new':
```

Comment:
I'd actually like for this to replace the existing tfoda implementation. Perhaps, a separate PR would be better?